Makefile

```diff
@@ -237,6 +237,9 @@ run-local-avl-subscriptions:
 run-local-avl-subscription:
 	STAGE=local TABLE_NAME=${AVL_SUBSCRIPTION_TABLE_NAME} AVL_PRODUCER_API_KEY_ARN=${AVL_PRODUCER_API_KEY_ARN} SUBSCRIPTION_ID="${SUBSCRIPTION_ID}" npx tsx -e "import {handler} from './src/functions/avl-subscriptions'; handler({ pathParameters: { subscriptionId: '${SUBSCRIPTION_ID}' }}).then(console.log).catch(console.error)"
 
+run-local-avl-validate:
+	STAGE=local npx tsx -e "import {handler} from './src/functions/avl-validate'; handler({ body: '\{\"url\":\"http://ee7swjlq51jq0ri51nl3hlexwdleoc8n.lambda-url.eu-west-2.localhost.localstack.cloud:4566\",\"username\":\"test-user\",\"password\":\"dummy-password\"\}' }).then(console.log).catch(console.error)"
```

Comment:
I presume the provided localstack URL will not persist, in which case can you add a comment about how to run locally against such a URL?

Reply:
I think in this case I'll remove the local make command - it doesn't really add much value to run locally like this. To test this function locally you can spin it up in localstack and run the `make command-invoke-avl-validate` command and enter the details of an operator to test the response.